services/headless-lms/server/src/controllers/course_material/pages.rs

```diff
@@ -68,12 +70,24 @@ async fn get_url_path(
     Ok(page.url_path)
 }
 
+#[generated_doc]
+#[instrument(skip(pool))]
+async fn is_chapter_front_page(
+    page_id: web::Path<Uuid>,
+    pool: web::Data<PgPool>,
+) -> ControllerResult<web::Json<IsChapterFrontPage>> {
+    let mut conn = pool.acquire().await?;
+    let is_chapter_front_page = models::pages::is_chapter_front_page(&mut conn, *page_id).await?;
+    Ok(web::Json(is_chapter_front_page))
+}
+
 pub fn _add_routes(cfg: &mut ServiceConfig) {
     cfg.route("/exam/{page_id}", web::get().to(get_by_exam_id))
         .route("/{current_page_id}/next-page", web::get().to(get_next_page))
         .route("/{current_page_id}/url-path", web::get().to(get_url_path))
         .route(
             "/{current_page_id}/chapter-and-course-information",
             web::get().to(get_chapter_and_course_information),
-        );
+        )
+        .route("/{current_page_id}", web::get().to(is_chapter_front_page));
```

Comment:
maybe "/{current_page_id}/is_front_page", otherwise would assume endpoint returns all page data

Reply:
Okay I will make the change